doc/plugin/EEPROM-Settings.md

```diff
@@ -64,8 +64,18 @@ The plugin provides the `EEPROMSettings` object, which has the following methods
 > keyboard boots up, it will automatically switch to the configured layer - if
 > any.
 >
-> This is the Focus counterpart of the `default_layer()` method documented
-> above.
+> Setting it to `126` or anything higher disables the automatic switching.
+
+### `use_eeprom_layers_only([true|false])`
```

Comment:
perhaps "ignoreHardcodedLayers"?

Reply:
`ignoreBuiltInLayers`? or `ignoreDefaultLayers`? We refer to them as default layers elsewhere - not sure if that's a good thing, but however we refer to them, adjusting this name to match would be best I think.

Reply:
"Default" feels funny to me. I'm ok with "BuiltIn", but since I' usually spell it "Builtin", I'm not in love with it. IgnoreFirmwareLayers? IgnoreCompiledLayers?

Reply:
`IgnoreBuiltinLayers` works for me, not attached to capitalizing that `I`. `IgnoreCompiledLayers` would be my second choice, `IgnoreFirmwareLayers`.... makes me feel uneasy, not quite sure why.

Reply:
(My worry about capitalization and that I is that it'll add cognitive load remembering which way we have it, rather than preferring any specific version of it.)

Hardcoded doesn't do it for you?

Reply:
It beats the name I gave the method, but it doesn't feel right to me. I don't have any better suggestions however, so we can go with `IgnoreHardcodedLayers`. It's not very important, because the primary user of this is `EEPROMKeymap`, and it presents it under a different name (`keymap.useCustom`). IOW, it's not user-facing in the majority of cases.